tests/system/runtimes/test_databricks.py

```diff
@@ -67,20 +89,12 @@ def _add_databricks_env(function, is_cluster_id_required):
         for name, val in job_env.items():
             function.spec.env.append({"name": name, "value": val})
 
-    @pytest.mark.parametrize(
-        "use_existing_cluster, fail", [(True, False), (False, True), (False, False)]
-    )
+    @pytest.mark.parametrize("use_existing_cluster, fail", [(True, False)])
```

Comment:
Is this an accidental change? There's no point of parameterizing if we only test with one set of parameters.